### PR TITLE
Monitoring db naming

### PR DESCRIPTION
**Pull request by ConnorPigg**
Closes issue #564 and changes time-stamping to avoid string formatting to preserve precision. Pushes formating to UI.

### CODE REVIEW

File: parsl/dataflow/dflow.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ def __init__(self, config=Config()):
         workflow_info = {
                 'python_version': sys.version_info,
                 'parsl_version': get_version(),
-                "time_began": str(self.time_began.strftime('%Y-%m-%d %H:%M:%S')),
+                "time_began": str(self.time_began),
```
Comment (annawoodard) [inline parsl/dataflow/dflow.py]: Can we make float columns be `sqlalchemy.Float` type? Then we don't need to convert this to a string.

File: parsl/dataflow/dflow.py [hunk 2]
```diff
@@ -409,7 +409,7 @@ def launch_task(self, task_id, executable, *args, **kwargs):
             executable = app_monitor.monitor_wrapper(executable, task_id, self.monitoring_config, self.run_id)
         exec_fu = executor.submit(executable, *args, **kwargs)
         self.tasks[task_id]['status'] = States.running
-        self.tasks[task_id]['time_started'] = str(datetime.now().strftime('%Y-%m-%d %H:%M:%S'))
+        self.tasks[task_id]['time_submitted'] = time.time()
```
Comment (annawoodard) [inline parsl/dataflow/dflow.py]: Can we move this `time.time()` call to the very start of this method? In my view, `time_submitted` should be the time the _user_ submitted the task. For measuring latency we want to be able to measure how long it takes us to go round trip (user submits task -> user receives task output).

File: parsl/dataflow/dflow.py [hunk 3]
```diff
@@ -280,7 +280,7 @@ def handle_update(self, task_id, future, memo_cbk=False):
                 final_state_flag = True
                 self.tasks_failed_count += 1
 
-                self.tasks[task_id]['time_completed'] = str(datetime.now().strftime('%Y-%m-%d %H:%M:%S'))
+                self.tasks[task_id]['time_completed'] = time.time()
```
Comment (annawoodard) [inline parsl/dataflow/dflow.py]: I vote to change `time_completed` to `time_returned`, to distinguish when the task finished running on the worker from when the task was returned to the DFK. Soon we should add a `time_completed` which will refer to when the task finished running on the worker.

File: parsl/dataflow/dflow.py (hunk 1 above)
Comment (ConnorPigg) [reply parsl/dataflow/dflow.py]: Currently, all log information is being converted to strings by the handler that sends the log over HTTP inside db_local.py RemoteHandler. I convert them to strings here to be explicit about this. The reason for this conversion had to do with a problem while sending the information over HTTP.

File: parsl/dataflow/dflow.py (hunk 3 above)
Comment (ConnorPigg) [reply parsl/dataflow/dflow.py]: That makes sense and I will change this in the code.

File: parsl/dataflow/dflow.py (hunk 1 above)
Comment (annawoodard) [reply parsl/dataflow/dflow.py]: Got it, thanks for the clarification!

**Pull request merged**